lighthouse-core/test/audits/byte-efficiency/duplicated-javascript-test.js

```diff
@@ -85,156 +85,86 @@ describe('DuplicatedJavascript computed artifact', () => {
           Object {
             "source": "Control/assets/js/vendor/ng/select/select.js",
             "sourceBytes": Array [
-              48513,
-              48513,
+              16009,
```

Comment:
hm this feels weird to say that the `sourceBytes` are transferSize, no? 

I agree with wasted computations using transferSize, that seems great 👍 (based on what was happening with `wastedBytesByUrl` I assumed they already were, and looks like that was half true and we were only using that for the opportunity savings and not the displayed wasted bytes?)

Reply:
> hm this feels weird to say that the sourceBytes are transferSize, no?

`sourceTransferBytes`? I'm wondering if in the long run we'd want to modify the bytes type: `{transfer: ..., resource: ...}`.

> based on what was happening with wastedBytesByUrl I assumed they already were, and looks like that was half true and we were only using that for the opportunity savings and not the displayed wasted bytes

yea, that map is only for the base class to calculate savings in latern. only showed up as the seconds savings estimate in the report.

Reply:
sourceTransferBytes sgtm 👍

Reply:
actually, given that `wastedBytes` and `totalBytes` on all opportunities are in terms of transfer size, I'm hesitant to break consistency in naming here. We don't specify transfer or resource bytes in any of our byte keys.

why does it feel weird?

Reply:
it feels weird here because we're specifically talking about the source code, unlike in all the other opportunities that are always talking about what was transferred. the introduction of source map data changes the game IMO because there's a completely new size we could be talking about, the original source code size. Calling something `sourceSize` makes me think it's the size of that original source code rather than the bytes that were transferred corresponding to a subset of that source code.